src/coredata.cs

```diff
@@ -2581,5 +2664,79 @@ interface NSPersistentCloudKitContainerOptions {
 		[Export ("initWithContainerIdentifier:")]
 		[DesignatedInitializer]
 		IntPtr Constructor (string containerIdentifier);
+
+		[Watch (7, 0), TV (14, 0), Mac (11, 0), iOS (14, 0)]
+		[Export ("databaseScope", ArgumentSemantic.Assign)]
+		CKDatabaseScope DatabaseScope { get; set; }
 	}
+
+	[Watch (7,0), TV (14,0), Mac (11,0), iOS (14,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface NSPersistentCloudKitContainerEvent : NSCopying {
+		[Export ("identifier", ArgumentSemantic.Strong)]
+		NSUuid Identifier { get; }
+
+		[Export ("storeIdentifier", ArgumentSemantic.Strong)]
+		string StoreIdentifier { get; }
+
+		[Export ("type")]
+		NSPersistentCloudKitContainerEventType Type { get; }
+
+		[Export ("startDate", ArgumentSemantic.Strong)]
+		NSDate StartDate { get; }
+
+		[NullAllowed, Export ("endDate", ArgumentSemantic.Strong)]
+		NSDate EndDate { get; }
+
+		[Export ("succeeded")]
+		bool Succeeded { get; }
+
+		[NullAllowed, Export ("error", ArgumentSemantic.Strong)]
+		NSError Error { get; }
+
+		[Notification]
+		[Watch (7, 0), TV (14, 0), Mac (11, 0), iOS (14, 0)]
+		[Field ("NSPersistentCloudKitContainerEventChangedNotification")]
+		NSString ChangedNotification { get; }
+
+		[Watch (7, 0), TV (14, 0), Mac (11, 0), iOS (14, 0)]
+		[Field ("NSPersistentCloudKitContainerEventUserInfoKey")]
+		NSString UserInfoKey { get; }
+	}
+
+	[Watch (7,0), TV (14,0), Mac (11,0), iOS (14,0)]
+	[BaseType (typeof (NSPersistentStoreRequest))]
+	interface NSPersistentCloudKitContainerEventRequest {
+		[Export ("resultType", ArgumentSemantic.Assign)]
+		NSPersistentCloudKitContainerEventResultType ResultType { get; set; }
+
+		[Static]
+		[Export ("fetchEventsAfterDate:")]
+		NSPersistentCloudKitContainerEventRequest FetchEvents (NSDate date);
```

Comment:
Minimally the parameter name should be `afterDate`, otherwise that information is lost

but I think it might be better to have `FetchEventsAfter (NSDate date)`  since `...beforeDate:` could be added later and cause confusion